src/index.js

```diff
@@ -148,14 +149,13 @@ async function bootstrap(state) {
 
 async function moveToProduction() {
   log.info('🚚 starting move to production');
-  await client.copyIndex(c.indexName, c.bootstrapIndexName, [
+
+  const currentState = await stateManager.get();
+  await client.copyIndex(c.bootstrapIndexName, c.indexName, [
```

Comment:
I believe we should write `await client.copyIndex(c.bootstrapIndexName, c.indexName)` which will copy the data + all settings (+synonyms, query rules).

The current line will not copy the data

Reply:
oh you're right, good of you for catching that!